src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -123,6 +123,7 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
       this.isSubmitting = true;
       this.error = null;
       let currentPageFields = this.formValueService.filterCurrentPageFields(this.currentPage.case_fields, this.editForm.value);
```

Comment:
can you please rename `currentPageFields` to `pageFormFields`

Reply:
applied changes